sessions.go

```diff
@@ -71,18 +71,19 @@ func loadSession(path string) *session {
 
 // lookupXdgDataDirs returns a string slice of all XDG_DATA_DIRS
 func lookupXdgDataDirs() []string {
+	var dirs []string
 	dataHome := os.Getenv("XDG_DATA_HOME")
```

Comment:
I think this value is now ignored...?

Reply:
line 80

Reply:
Line 80 is only reached of the `dataHome` is "", so I think the non-empty value is ignored...

Reply:
Thanks for addressing this, but I think it would need to be in an `else` block - otherwise a `dataHome` that is "" and then `err != nil` from `UserHomeDir` then you could still have the empty dir added to the list like earlier...